src/Plugin/Field/FieldWidget/ReactParagraphs.php

```diff
@@ -231,14 +233,15 @@ public function massageFormValues(array $values, array $form, FormStateInterface
    * @param int|null $entity_id
    *   Entity id if its existing.
    *
-   * @return \Drupal\Core\Entity\ContentEntityInterface
+   * @return \Drupal\react_paragraphs\Entity\ParagraphsRowInterface
    *   Row entity with set field values.
    *
    * @throws \Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException
    * @throws \Drupal\Component\Plugin\Exception\PluginNotFoundException
    * @throws \Drupal\Core\Entity\EntityStorageException
    */
-  protected function getRowEntity(array $field_data, $entity_id) {
+  protected function getRowEntity(array $field_data, ?int $entity_id): ParagraphsRowInterface {
```

Comment:
Same with these. Can we get our linters to check for them?

Reply:
yeah It would be nice to have some tool look for those. Not sure if codeclimate or maybe github actions will have something for that. :shrug: